bookwyrm/templates/user/layout.html

```diff
@@ -1,11 +1,13 @@
 {% extends 'layout.html' %}
-{% load i18n %}
-{% load humanize %}
-{% load utilities %}
-{% load markdown %}
+{% load i18n %}{% load humanize %}{% load utilities %}{% load markdown %}{% load layout %}
```

Comment:
(Merging lines will increase the chance of conflicts when merging branches.)

Reply:
Ah yes, I wanted to keep the same format as the other templates I modified in this PR

Reply:
sorry that's my bad - I've been inconsistent about stacking these vs putting them on one line